.rubocop.yml

```diff
@@ -31,6 +31,7 @@ Style/AsciiComments:
 Metrics/LineLength:
   Max: 100
 Metrics/MethodLength:
+  Max: 20
```

Comment:
Please add it inline to the prams thing, because, globally i don't want methods that long